src/pages/Graph/SummaryPanelNodeTraces.tsx

```diff
@@ -128,7 +133,7 @@ class SummaryPanelNodeTraces extends React.Component<Props, State> {
   }
 
   render() {
-    const tracesDetailsURL = `/namespaces/${this.props.namespace}/applications/${this.props.service}?tab=traces`;
+    const tracesDetailsURL = `/namespaces/${this.props.namespace}/services/${this.props.app}?tab=traces`;
```

Comment:
this has changed, the correct one is now with `applications`

Reply:
whoops, merge issue.  Assumingyou fixed after the merge.